### PR TITLE
Removed Gson dependency from server and client

### PR DESCRIPTION
**Pull request by kpiwko**
* Replaced with JSONObject and JSONArray custom parsing

### CODE REVIEW

**Comment by lukeis** [comment]
it shouldn't be necessary to remove it from selendroid-client and it's nicer to use the more compact library there

File: selendroid-server-common/src/main/java/io/selendroid/server/common/utils/CallLogEntry.java [hunk 1]
```diff
@@ -32,6 +48,38 @@ public CallLogEntry(String number, int duration, Date time, int direction) {
...
+    public String toJSON() {
+        try {
+            JSONObject json = new JSONObject();
+            json.put("number", number);
+            json.put("duration", duration);
+            json.put("direction", direction);
+            json.put("time", dateFormat.format(time));
+            return json.toString();
+        }
+        catch(JSONException e) {
```
Comment (asm89) [inline selendroid-server-common/src/main/java/io/selendroid/server/common/utils/CallLogEntry.java]: Can you put the catch on the line above? `} catch(...) {` in other places as well.

**Comment by asm89** [comment]
Can you fix the coding style? Thank you for fixing this. :+1:

**Comment by DominikDary** [comment]
:+1:

**Comment by kpiwko** [comment]
Sorry, github notifications were filter to wrong folder. Coding style was fixed.

**Comment by kpiwko** [comment]
@lukeis I removed gson from the client as well to be sure that same code is used to marshal and demarshal JSON. For instance, with Date object it might be complicated to get it in GSON expected format from the custom utility.

**Pull request merged**